### DISCUSSION TITLE
What is this badge? "contributors-anon"

### DISCUSSION THREAD

**Discussion by MichaelHinrichs** [badges/shields#10322] [Q&A]
I found this badge in an old readme, but i cant find anything like it on shields.io. It's not a static badge, but there's no page for it. The only badges for github contributers use allcontributors, but this is something else. What is contributors-anon?
![GitHub repo contributors](https://img.shields.io/github/contributors-anon/MichaelHinrichs/GitHub-Actions?style=flat&logo=github)
`https://img.shields.io/github/contributors-anon/:user/:repo`

**Comment by chris48s** [comment]
There are 2 variants of the GitHub Contributors badge.

- ![](https://img.shields.io/github/contributors/badges/shields) - https://img.shields.io/github/contributors/badges/shields
- ![](https://img.shields.io/github/contributors-anon/badges/shields) - https://img.shields.io/github/contributors-anon/badges/shields

The difference is that the `-anon` variant includes anonymous contributors, whereas the standard one excludes them.

An anonymous contribution happens when a commit is made and the email address on the commit is not linked to any GitHub account. You've probably seen them, but here's an example:

![Screenshot at 2024-07-04 20-57-48](https://github.com/badges/shields/assets/6025893/30c95410-175e-49d9-be1c-ea30baffbe1c)

You've also raised another point which is that neither of these have a documentation page on https://shields.io/ I've raised an issue about fixing this at https://github.com/badges/shields/issues/10323
